platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -993,7 +999,13 @@ def get_lpmode(self):
         """
         lpmode_state = False
         try:
-            if self.sfp_type.startswith('QSFP'):
+            if self.sfp_type == 'QSFP_DD':
```

Comment:
To have proper value of sfp_type, you need to port _initialize_media() from e_sonic.